javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,51 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1978;
+let yearFuture = 2045;
+let age = yearFuture - yearOfBirth;
+console.log ('You will be ' + age + ' years old in ' + yearFuture + ' ');
```

Comment:
Looks good. But maybe look into when to use const vs let

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#description
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let